_config.php

```diff
@@ -1,3 +1,3 @@
 <?php
-define("COLOURPICKER_DIR", "colourpicker");
+define("COLOURPICKER_DIR", "ninty9notout/colourpicker");

```

Comment:
This isn't a directory, no real need to use it, better to just use `ninty9notout/colourpicker` where needed